index.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+alert("First number is larger than Second number");
+
+let firstNumber = prompt("Enter first number");
+let secondNumber = prompt("Enter second number");
+
+let compare;
+
+if (firstNumber > secondNumber) {
+    alert(compare = "I am bigger");
+} else {
+    alert(compare = "I am smaller");
+}
```

Comment:
Here you used simple condition statement if() then {} else {} which covers properly only (first > second) condition.
To cover "less" than and "equal" conditions you need to do complex condition statement if() then {} else if()...